components/CreateAlertModal.tsx

```diff
@@ -258,6 +259,21 @@ const CreateAlertModal: FunctionComponent<CreateAlertModalProps> = ({
     actions.loadAlerts(mangoAccount.publicKey)
   }, [])
 
+  // Delete notifi Alerts that have fired
+  useEffect(() => {
+    const firedAlert = data?.alerts.find((item) => {
+      const filterOptions = item.filterOptions ?? '{}'
+      const parsedOptions = JSON.parse(filterOptions)
+      return parsedOptions.delayProcessingUntil === '12/31/9999 23:59:59'
```

Comment:
Is it possible to return a boolean like `isObsolete` ?

Reply:
That's a good question, it could be interesting to expose helper methods on `Alert` somehow rather than expecting clients to parse this way

Reply:
moved this logic into the SDK's `isAlertObsolete` utility function